mkdocs.yml

```diff
@@ -254,7 +254,9 @@ nav:
       - Webhooks: "user-guide/platform-functionality/webhook.md"
   - Developer Guide:
     - "development/index.md"
-    - Jobs Developer Guide: "development/jobs.md"
+    - Jobs Developer Guide:
+      - "development/jobs.md"
+      - Migration Guides: "development/migration/index.md"
```

Comment:
Actually, I'm reconsidering this - since `development/jobs.md` already contains a link to `user-guide/administration/upgrading/from-v1/migrating-jobs-from-nautobot-v1.md` this general migration guide is out of place here, plus it looks like having the same file at two places in the TOC messes up the navigation sidebar a bit. Perhaps instead?

- move `user-guide/administration/upgrading/from-v1/migrating-jobs-from-nautobot-v1/` to `development/jobs/migration/from-v1.md`
  - Add a link from this file to the `development/migration/index.md` indicating that that file provides general code migration information, while this file adds Job-specific information.
  - Also add a link from `development/migration/index.md` back to this file with appropriate text.
- move `development/jobs.md` to `development/jobs/index.md` to be consistent with the other development subdirectories.
- TOC here would then be:
    ```
    - Jobs Developer Guide:
      - "development/jobs/index.md"
      - Migration Guides:
        - "development/jobs/migration/from-v1.md"
  ```

Does that sound reasonable?